av1-rtp-spec.md

```diff
@@ -199,6 +199,13 @@ previous OBU, 0 otherwise
 Y: set to 1 if the last OBU contained in the packet will continue in another
 packet, 0 otherwise
 
+X: if set to 0, each OBU (or OBU fragment) MUST be preceded by a length

```

Comment:
As discussed in the last meeting, this is not sufficient. The parser won't know which fragment is the last and therefore won't know whether it should attempt to read a length field.

Reply:
Fixed.